signac/core/h5store.py

```diff
@@ -215,17 +215,30 @@ def __eq__(self, other):
 
 
 class H5Store(MutableMapping):
-    """An HDF5 backend for storing arbitrary array-like and dictionary-like data.
-    Both attribute-based and index-based operations are supported.
+    """An HDF5-backed container for storing array-like and dictionary-like data.
+
+    The H5Store API is a :class:`collections.abc.MutableMapping` and therefore
+    behaves similar to a :class:`dict`, but all data is stored persisently in
+    the associated HDF5-file on disk.
+
+    Supported types include:
+
+      * built-in types (int, float, str, bool, NoneType, array)
+      * numpy arrays
+      * pandas data frames (requires pandas and pytables),
+
+    as well as mappings of values of these types. Values can be accessed as
+    attributes (`h5s.foo`) or via key index (`h5s['foo']`).
```

Comment:
@csadorf Did you render the "code font" change? I think that using one backtick doesn't work, I believe you must use one of the following:
```
:code:`h5s.foo`
```
or
```
``h5s.foo``
```